support/index.html

```diff
@@ -126,7 +126,7 @@ <h1 class="navbar-brand">
         usernameStyle: 'email',
         defaultDatabaseConnection: 'acme',
         prefill: {
-          email: 'johnfoo@gmail.com'
+          email: 'luis@luisrudge.net'
```

Comment:
Probably going to get a lot more spam from web crawlers...  But then johnfoo might appreciate it.

Reply:
fixed